packages/editor/src/components/commands/index.js

```diff
@@ -261,10 +261,22 @@ function useEditorCommandLoader() {
 }
 
 function useEditedEntityContextualCommands() {
-	const { postType } = useSelect( ( select ) => {
-		const { getCurrentPostType } = select( editorStore );
+	const { postType, isViewable, status, link } = useSelect( ( select ) => {
+		const {
+			getCurrentPostType,
+			getEditedPostAttribute,
+			getEditedPostPreviewLink,
+		} = select( editorStore );
+		const { getPostType } = select( coreStore );
+		const isPublished = getEditedPostAttribute( 'status' ) === 'publish';
```

Comment:
Tiny semantic nitpick: the labels "Preview post" and "Preview page" seem fine to me for most statuses that are not publish, e.g., draft, pending, scheduled. 

There's another named "Private", which is not in a pre-publish state, but rather "published" for a restricted set of users, usually admin and editor.

Not a blocker for this PR, just thinking out aloud. 😄

Reply:
So in that case, what will you suggest for label. 

Should we go with `View`?

More suggestions are welcome.

Reply:
I think we can keep the labels as you have them for this PR. I don't want to block momentum because of a label.

It can always be a follow up later if folks decide it's worth doing. 👍🏻